flocker/docs/test/test_version.py

```diff
@@ -0,0 +1,153 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Test for :module:`flocker.docs.version`.
+"""
+
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from .._version import (
+    parse_version, FlockerVersion,
+    get_doc_version, is_release,
+)
+
+
+class ParseVersionTests(SynchronousTestCase):
```

Comment:
Add tests (or create an issue for adding tests) for the other types of release (marketing, pre)